src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -399,32 +410,44 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                                 Action<SourceCacheContext> cacheModifier = (cache) => { };
 
                                 var isRestoreOriginalAction = true;
-                                var restoreSummaries = await DependencyGraphRestoreUtility.RestoreAsync(
-                                    _solutionManager,
-                                    dgSpec,
-                                    cacheContext,
-                                    providerCache,
-                                    cacheModifier,
-                                    sources,
-                                    _nuGetProjectContext.OperationId,
-                                    forceRestore,
-                                    isRestoreOriginalAction,
-                                    additionalMessages,
-                                    l,
-                                    t);
-
-                                _packageCount += restoreSummaries.Select(summary => summary.InstallCount).Sum();
-                                var isRestoreFailed = restoreSummaries.Any(summary => summary.Success == false);
-                                _noOpProjectsCount += restoreSummaries.Where(summary => summary.NoOpRestore == true).Count();
-                                _solutionUpToDateChecker.SaveRestoreStatus(restoreSummaries);
-
-                                if (isRestoreFailed)
+                                var isRestoreFailed = false;
```

Comment:
I know the old prop was `isRestoreFailed`, but in general, it'd be nice if we stuck to positive names to improve readability :)

Reply:
Renamed to `isRestoreSucceeded`